.github/workflows/monitoring.yml

```diff
@@ -26,5 +26,5 @@ jobs:
           go-version: 1.17
       - name: Test
         run: go test -count=1 -race -v ./pkg/monitoring/...
-      - name: Bench
-        run: go test -count=1 -bench=. ./pkg/monitoring/...
+      #- name: Bench
```

Comment:
is this on propuse?

Reply:
Yeah, they crash on CI
I'll fix this in a separate PR!